src/utils/type.utils.ts

```diff
@@ -87,3 +87,18 @@ export const isNull = (value: any): value is null => {
 export const isFunction = (value: any): value is Function => {
   return typeof value === 'function';
 };
+
+/**
+ * Type check for array of numbers
+ *
+ * @param {any} value Any value
+ * @return {boolean} True if it is a array of numbers, false otherwise
+ */
+export const isArrayOfNumbers = (value: any): value is boolean => {
+  if (!isArray(value)) {
+    return false;
+  }
+
+  const isNotNumber = value.some((element) => !isNumber(element));
+  return !isNotNumber;
```

Comment:
```suggestion
  return value.some((element) => !isNumber(element));
```

This is pretty much clear enough where you don't have any benefit from an additional variable. If this was a more complex function, then the named variable would be a good choice for better readability.